packages/web-components/src/components/va-button-pair/va-button-pair.tsx

```diff
@@ -72,26 +72,31 @@ export class VaButtonPair {
 
   private handlePrimaryClick = (e: MouseEvent) => {
     this.primaryClick.emit(e);
-    if (this.disableAnalytics) return;
-    this.componentLibraryAnalytics.emit({
-      componentName: 'va-button-pair',
-      action: 'click',
-      details: {
-        // TODO: add analytics event details
-      },
-    });
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-button-pair',
+        action: 'click',
+        details: {
+          type: 'primary',
+        },
+      };
+      this.componentLibraryAnalytics.emit(detail);
```

Comment:
This seems like it might be redundant considering the `va-button` will be emitting its own event as well and the `button-type` value will either be `primary` or `secondary`. Do we need the pair to emit its own event?

Reply:
The analytics ticket wanted an event sent for `va-button-pair` which I think the reason is so that GA knows `va-button-pair` was used. I don't know the answer to if we should remove this because of a `va-button` redundancy. I would lean towards having unique analytics events for individual components just based on my very limited knowledge of GA (even if those components contain children). I am not sure if this will cause "double logging" in GA though or if that would even be an issue with them.

Reply:
It would cause double logging because we don't stop the analytics event for the button from propagating.

Reply:
@michelle-dooley We sometimes use our own design system components in our other design system components. For example in this case, we are using the `va-button` inside of `va-button-pair` and there are separate GA events for those components: `int-button-pair-click` and `cta-button-click`.

So when `va-button` is clicked, both `int-button-pair-click` and `cta-button-click` fire. This _seems_ like it would be an issue for accurate reporting in GA but I just wanted to confirm with you before we dug into a technical solution on our end. The solution may end up being that we remove the GA event for `va-button-pair` completely and just rely on the `va-button` event itself.

Reply:
@bkjohnson I think after the [verification by Michelle](https://github.com/department-of-veterans-affairs/va.gov-team/issues/45510#issuecomment-1320211727) that we would be seeing double logging here and it would be a bit redundant as you mentioned originally, I'm ok with removing this from `va-button-pair` if you're comfortable with that too.

Reply:
We could stop the propagation of the analytics event coming from the button and just let the one from the button pair bubble up.

Reply:
How would we know when to stop it? Would we be checking if `va-button-group` is a parent element of `va-button` from within `va-button`?

Reply:
We would know when to stop it if the `<va-button-pair>` receives an analytics event bubbling up from its button children.

Reply:
@bkjohnson I think I'm on the same page now. I've updated va-radio-button to handle that analytics events like this: https://github.com/department-of-veterans-affairs/component-library/pull/513/commits/87a38facab601f4902c6584eb0ddd050425f3d34